core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -172,6 +172,15 @@ function totalDurationColor(totalGCTime, totalDuration) {
     return (totalGCTime > GCTimePercent * totalDuration) ? "white" : "black";
 }
 
+function memoryUsageTooltip(onHeap, maxOnHeap, offHeap, maxOffHeap, onHeapSum, offHeapSum, type) {
```

Comment:
`onHeapSum` and `offHeapSum` mean something completely different, right?  aren't they more like `totalMemoryUsed` and `totalMemoryAvailable`?